lib/components/Actions/Actions.tsx

```diff
@@ -1,34 +1,23 @@
-import React, { Component, Children, ReactNode } from 'react';
+import React, { Children, ReactNode } from 'react';
 import { ActionsProvider } from './ActionsContext';
-import Box from '../Box/Box';
+import { Box } from '../Box/Box';
 
 export interface ActionsProps {
   children: ReactNode;
 }
 
-export default class Actions extends Component<ActionsProps> {
-  static displayName = 'Actions';
-
-  render() {
-    const { children } = this.props;
-
-    return (
-      <ActionsProvider value={true}>
-        <Box display="flex" flexDirection={['column', 'row']}>
-          {Children.map(children, (child, index) =>
-            index === 0 ? (
-              <div>{child}</div>
-            ) : (
-              <Box
-                paddingLeft={['none', 'xsmall']}
-                paddingTop={['xsmall', 'none']}
-              >
-                {child}
-              </Box>
-            ),
-          )}
-        </Box>
-      </ActionsProvider>
-    );
-  }
-}
+export const Actions = ({ children }: ActionsProps) => (
```

Comment:
Probably a bit more global, but shouldn't this read something like:

```tsx
export const Actions: FunctionComponent<ActionsProps> = ({ children }) => {};
```

so; type the lambda, rather than just the props? It'll alleviate the need to actually say that "ActionsProps" contains "children", like you've done on line 5.